Angular/src/app/add-capability/add-capability.component.html

```diff
@@ -0,0 +1,47 @@
+<div class="modal-header">
+    <h5 class="modal-title" id="exampleModalLabel">Add Capability</h5>
+    <button
+      type="button"
+      class="close"
+      aria-label="Close"
+      (click)="activeModal.dismiss()"
+    >
+      x
+    </button>
+  </div>
+  
+  <form [formGroup]="addCapabilityForm" (ngSubmit)="onSubmit()" autocomplete="off">
+    <div class="modal-body" data-keyboard="false" data-backdrop="static">
+      <div class="form-group">
+        <label for="capabilityName">Capability Name</label>
+        <input
+          type="text"
+          formControlName="capabilityName"
+          class="form-control"
+          [ngClass]="{ 'is-invalid': submitted && formControls.capabilityName.errors }"
+        />
+        <div
+          *ngIf="submitted && formControls.capabilityName.errors"
+          class="invalid-feedback"
+        >
+          <div *ngIf="formControls.capabilityName.errors.required">
+            Capability Name is a required field
+          </div>
```

Comment:
Just always watch the indentation/formatting of your HTML. It seems minor, but it's important for readability. This `div` should be decremented back one